package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@storyblok/mui",
   "description": "A Storyblok MUI theme with reusable components.",
-  "version": "0.0.21",
+  "version": "0.0.22-beta.1",
```

nothing important, but it normally starts with 0
```suggestion
  "version": "0.0.22-beta.0",
```